Formula/cvc4.rb

```diff
@@ -30,6 +30,9 @@ def install
       args << "CXXFLAGS=-I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/JavaVM.framework/Versions/A/Headers/"
     end
 
+    args << "--enable-gpl" if build.with? "readline"
```

Comment:
Another approach would be to add a separate flag for `--enable-gpl`, so that options passed to `brew` would align more closely with those passed to `./configure`